src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__review-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  .stars {
+    display: flex;
+    align-items: center;
+
+    &--4 {
+      display: flex;
+    }
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-position: center;
+      background-repeat: no-repeat;
+      background-image: url(../images/star.svg); // Inactive star
+
+      // Active stars based on rating
+      &:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
```

Comment:
The `background-image` property here overrides the one defined on line 20. To follow the DRY (Don't Repeat Yourself) principle, consider creating a modifier class for the active star that can be applied to the star elements instead of using `nth-child`. This approach will make it easier to manage the number of active stars dynamically with JavaScript if needed in the future.